### PR TITLE
Add getReefNameAndRegion  helper

### PR DESCRIPTION
**Pull request by ericboucher**
This helps make sure that we can use the display name for the search tool.

### CODE REVIEW

File: packages/website/src/common/Search/index.tsx [hunk 1]
```diff
@@ -13,18 +13,34 @@ import Autocomplete from "@material-ui/lab/Autocomplete";
...
 import type { Reef } from "../../store/Reefs/types";
 import { reefsListSelector } from "../../store/Reefs/reefsListSlice";
+import { formatReefName } from "../../store/Reefs/helpers";
 
 const Search = ({ classes }: SearchProps) => {
   const [searchedReef, setSearchedReef] = useState<Reef | null>(null);
   const dispatch = useDispatch();
-  const reefs = useSelector(reefsListSelector).filter(({ name }) => name);
+  const reefs = useSelector(reefsListSelector)
+    .filter((reef) => formatReefName(reef))
```
Comment (SimplyBallistic) [inline packages/website/src/common/Search/index.tsx:22]:
```suggestion
    .filter(formatReefName)
```
I think you can just pass the function reference here if you wish

File: packages/website/src/store/Reefs/helpers.ts [hunk 2]
```diff
@@ -4,6 +4,9 @@ import type { Reef } from "./types";
 import { degreeHeatingWeeksCalculator } from "../../helpers/degreeHeatingWeeks";
 import { alertFinder } from "../../helpers/bleachingAlertIntervals";
 
+export const formatReefName = (reef: Reef) =>
```
Comment (SimplyBallistic) [inline packages/website/src/store/Reefs/helpers.ts:7]: More of a personal opinion, but I feel `format` as a function verb more-so implies the string is edited in some way. Instead, what we're doing here is 'getting' the reef name, checking various sources. Do you think `getReefName` would be better here?

File: packages/website/src/common/Search/index.tsx [hunk 3]
```diff
@@ -13,18 +13,34 @@ import Autocomplete from "@material-ui/lab/Autocomplete";
...
+    // Sort by formatted name
+    .sort((a, b) => {
+      const nameA = formatReefName(a) || "";
+      const nameB = formatReefName(b) || "";
+
+      if (nameA > nameB) {
```
Comment (SimplyBallistic) [inline packages/website/src/common/Search/index.tsx:28]: Using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare might be better here as in the future, special characters could be used if the application is truly international.

**Review by tleraj — APPROVED** [review]

**Review by SimplyBallistic — COMMENTED** [review]
Looks good!
Edit: Didn't realize this was merged

**Pull request merged**